src/stores/messages/MessagesDataProviderStore.ts

```diff
@@ -387,4 +452,31 @@ export default class MessagesDataProviderStore {
 
 		return message;
 	};
+
+	@action
+	keepLoading = async (direction: 'next' | 'previous') => {
+		if (
+			this.messagesStore.filterStore.filter.streams.length === 0 ||
+			!this.searchChannelNext ||
+			!this.searchChannelPrev
+		)
+			return;
+
+		const query = {
```

Comment:
this query includes filters right now whether ifSoftFilter applied or not, and here we might want to resume soft channels also if they are stopped